### PR TITLE
ac-3: week-2:Fixed bug related to fetching semester and issues in template

### LINKED ISSUE #1007: Error while fetching Semester of student in Online_cms module
**Describe the bug**
Error while fetching Semester of student in Online_cms module

**To Reproduce**
login with 2020 batch roll no
go to http://127.0.0.1:8000/ocms/

### PR DESCRIPTION
**Pull request by tech-akash**
## Issue that this pull request solves

Closes:  #1007 and #1031

### Brief description of what is fixed or changed
Now users can access localhost:8000/ocms/ and after adding data to the database table (Register and Curriculum Instructor) they go to /ocms/<course_code in which student in currently enrolled in>/ to view that course data

## Types of changes

_Put an `x` in the boxes that apply_

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Other (please describe):

## Checklist

_Put an `x` in the boxes that apply_

-   [ ] My code follows the [style guidelines of this project](https://docs.google.com/document/d/1GI2Hile8UbGZ82gFe1y4wAVPcNPXip1cmXTzog1S41U/edit)
-   [ ] I have performed a self-review of my own code
-   [ ] I have created new branch for this pull request
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generate no new warnings
-   [ ] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by abhishekkumar5606** [comment]
LGTM

File: FusionIIIT/applications/online_cms/views.py [hunk 1]
```diff
@@ -81,86 +89,86 @@ def course(request, course_code):
...
+        #     v = requests.get(videos_url, params=videos_params)
+        #     results2 = v.json()['items']
+        #     videos = []
+        #     for res in results2:
+        #         video_data = {
+        #             'id': res['id'],
+        #             'title': res['snippet']['title'],
+        #         }
+
+        #         videos.append(video_data)
```
Comment (akshatnema) [inline FusionIIIT/applications/online_cms/views.py:171]: If you are commenting out the code and it is not needed in the future, kindly remove it.

File: FusionIIIT/applications/online_cms/views.py [hunk 2]
```diff
@@ -247,48 +255,49 @@ def course(request, course_code):
...
+        # v = requests.get(videos_url, params=videos_params)
+        # results2 = v.json()['items']
+        # videos = []
+        # for res in results2:
+        #     video_data = {
+        #         'id': res['id'],
+        #         'title': res['snippet']['title'],
+        #     }
+
+        #     videos.append(video_data)
```
Comment (akshatnema) [inline FusionIIIT/applications/online_cms/views.py:299]: Same here.

**Review by akshatnema — CHANGES_REQUESTED** [review]
Kindly resolve the merge conflicts.

**Pull request closed without merging**